### ISSUE TITLE
@int128_str error when using solve

### ISSUE THREAD

**Issue by elkaps** [chakravala/Reduce.jl#48]
Hello;
I'm having the following problem:
```
julia> 1.1111e-15*:a
:((11111a) / 10000000000000000000)

julia> solve(1.111e-15*:a)
(:(a = 0),)

julia> solve(1.1111e-15*:a)
ERROR: Reduce: Macro Core.var"@int128_str" block structure not supported

10000000000000000000

```
I'm using Reduce version v1.2.9

**Comment by chakravala** [comment]
What version of Julia?

**Comment by elkaps** [comment]
I'm using Julia v1.5.3

**Comment by chakravala** [comment]
The `Core.var"@int128_str"` parsing bug is now fixed in `v1.2.10` of the `Reduce` package.

**Comment by elkaps** [comment]
Thanks!
